Library/Homebrew/shims/mac/super/make

```diff
@@ -1,12 +1,4 @@
 #!/bin/bash
 
-# shellcheck disable=SC2154
-# HOMEBREW_CCCFG is set by extend/ENV/super.rb
```

Comment:
This comment seems to refer to `HOMEBREW_CCCFG` instead of `HOMEBREW_MAKE`. Is it correct for it to be removed?

Reply:
Same question! As long as `shellcheck` passes I'm fine with it being removed if it's no longer needed.

Reply:
I removed the comments because `brew style` with `shellcheck`  passed without them.

Reply:
Cool, just checking! :+1: